examples: add importable-configuration.nix

erikarvstedt opened the pull request:
Motivated by #395.

nixbitcoin reviewed (commented):
Concept ACK. @casey does this fulfill your use-case?

casey commented on examples/README.md:
```diff
@@ -17,19 +17,19 @@ By default, [`configuration.nix`](configuration.nix) enables `bitcoind` and `cli
 
+- [`./deploy-container-minimal.sh`](deploy-container-minimal.sh) creates a
+  container defined by [standalone-configuration.nix](standalone-configuration.nix).\
+  You can copy and import this file to use nix-bitcoin in an existing NixOS configuration.\
+  The configuration doesn't use the [secure-node.nix](../modules/presets/secure-node.nix) preset.\
+  Requires: [Nix](https://nixos.org/), a systemd-based Linux distro and root privileges
```
I think that if I were skimming paragraph, I might read the first sentence, think it only applied if you want to run `nix-bitcoin` in a container, and miss the rest.

So maybe it could be broken out into its own section:

```suggestion

-  You can copy and import this [standalone-configuration.nix](standalone-configuration.nix) to use nix-bitcoin in an existing NixOS configuration.\
  The configuration doesn't use the [secure-node.nix](../modules/presets/secure-node.nix) preset.\
  Requires: [Nix](https://nixos.org/), a systemd-based Linux distro and root privileges
```

casey reviewed (commented):
I added a comment suggestion breaking the item in `examples/README.md` in two to make it easier to notice while skimming.

It's not totally clear to me what "standalone" in `standalone-configuration.nix` refers to, maybe `nixos-configuration.nix` would be clearer?

casey commented:
> Concept ACK. @casey does this fulfill your use-case?

Yes, this definitely works for me.

jonasnick commented:
Looks good to me.

> It's not totally clear to me what "standalone" in standalone-configuration.nix refers to, maybe nixos-configuration.nix would be clearer?

Agree on the first point. Isn't it the opposite of "standalone" because it's supposed to be imported? Not sure if `nixos-configuration.nix` is more understandable because all configuration.nix files are NixOS configuration files. Alternatively `simple-configuration.nix`, `minimal-configuration.nix`, `importable-configuration.nix`?

erikarvstedt commented:
`importable-configuration.nix` is much better! I've renamed it.

Also, I've added a section `Get started` README section which directly points to the main use cases.

jonasnick reviewed (approved):
Liking the the README simplification, except that initally I would have preferred that the definition of a node had stayed (in particular since wallet does _not_ imply verification, unfortunately), but nix-bitcoin is more flexible now than "just" being a Bitcoin node.

ACK 2a16db69198d7a1b1143bf64f0480c5677ae5f75
